.travis.yml

```diff
@@ -7,6 +10,11 @@ php:
   - 5.6
   - hhvm
 
+ # cache composer cache across builds
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
it's not working currently, but maybe yes, Reason for skipping it was that you might get vendor dir from a completely different branch,so caching the cache seemed more safe.